oap-server/server-starter/src/main/resources/cilium-rules/exclude.yaml

```diff
@@ -0,0 +1,22 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+namespaces:
+    - kube-system
+
+labels:
+    - k8s:io.cilium.k8s.namespace.labels.istio-injection: enabled
+      k8s:security.istio.io/tlsMode: istio
```

Comment:
Does this rule mean `excluding istio relative services`? If so, I should consider whether this should be in default configurations.

Reply:
Yes, this configuration means excluding the services that have been added the proxy sidecar.

Reply:
Please add comments, and we should document this in the docs about cilium monitoring would be disabled automatically if services have been managed by mesh.

Reply:
updated, please check.